app/controllers/blog_posts_controller.rb

```diff
@@ -4,7 +4,7 @@ class BlogPostsController < ApplicationController
   before_filter :find_blog_post!, :only => :show
 
   def index
-    @blog_posts = BlogPost.all
+    @blog_posts = BlogPost.all.page(params[:page]).per(5)
```

Comment:
Fixed. No need to call ```.all```, and I set the number of posts per page to 10, in the ```config/initializers/kaminari_config.rb``` file